launcher/sdw_util/Util.py

```diff
@@ -163,3 +173,17 @@ def get_qt_version():
         return version
     else:
         raise ValueError("Qt version not supported: {}".format(version_str))
+
+
+def get_logger(prefix=SD_LOGGER_PREFIX, module=None):
+    if module is None:
+        return logging.getLogger(prefix)
+    else:
+        return logging.getLogger(prefix + "." + module)
+
+
+def strip_ansi_colors(str):
+    """
+    Strip ANSI colors from command output
+    """
+    return re.sub(r"\u001b\[.*?[@-~]", "", str)
```

Comment:
Open to a more robust solution here, of course, just don't want to write color codes to the log file.